content/docs/tasks/security/rbac-groups/index.md

```diff
@@ -0,0 +1,185 @@
+contains a JWT claim with the claim key being `groups` and the claim value being a list of strings [`"group1"`, `"group2"`].
+
+Set the environmental variable TOKEN to contain a valid sample JWT token:
+
+{{< text bash>}}
+$ TOKEN=$(curl {{< github_file >}}/security/tools/jwt/samples/groups-scope.jwt -s)
+{{< /text >}}
+
+With the authentication policy set, curl connection to the httpbin service should succeed if the JWT token is attached:
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+200
+{{< /text >}}
+
+curl connection to the httpbin service should fail if the JWT token is not attached:
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n"
+401
+{{< /text >}}
+
+### Configure groups-based authorization
+
+This section will create a policy to authorize the access to the `httpbin` service if the requests are
+originated from specific groups.
+
+First, Istio RBAC should be enabled for the namespace by running the following command:
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n $NS -f -
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: RbacConfig
+metadata:
+  name: default
+spec:
+  mode: 'ON_WITH_INCLUSION'
+  inclusion:
+    namespaces: ["rbac-groups-test-ns"]
+EOF
+{{< /text >}}
+
+As there may be some delays due to caching and other propagation overhead, wait a moment for the newly
+defined RBAC policy to become effective. Run the following command to verify that the curl connection
+to the httpbin service is rejected after the RBAC policy has become effective.
+
+{{< text bash >}}
+$ kubectl exec $(kubectl get pod -l app=sleep -n $NS -o jsonpath={.items..metadata.name}) -c sleep -n $NS -- curl http://httpbin.$NS:8000/ip -s -o /dev/null -w "%{http_code}\n" --header "Authorization: Bearer $TOKEN"
+403
+{{< /text >}}
+
+Next, create a `ServiceRole`, called httpbin-viewer, that allows read access to the `httpbin` service.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n $NS -f -
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: httpbin-viewer
+  namespace: rbac-groups-test-ns
+spec:
+  rules:
+  - services: ["httpbin.rbac-groups-test-ns.svc.cluster.local"]
+    methods: ["GET"]
+EOF
+{{< /text >}}
+
+Create a `ServiceRoleBinding` that assigns the `httpbin-viewer` role to users in the `group1`.
+
+{{< text bash >}}
+$ cat <<EOF | kubectl apply -n $NS -f -
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRoleBinding
+metadata:
+  name: bind-httpbin-viewer
+  namespace: rbac-groups-test-ns
+spec:
+  subjects:
+  - user: "*"
+    properties:
+      request.auth.claims[groups]: "group1"
```

Comment:
It might be worth to mention that currently `request.auth.claims` is the only property that supports matching against a `list of strings` in the JWT token as user may assume they can use list of strings in other properties.

Reply:
Done.